src/basic_recipes/contourf.jl

```diff
@@ -84,12 +84,12 @@ function Makie.plot!(c::Contourf{<:Tuple{<:AbstractVector{<:Real}, <:AbstractVec
     lowcolor = Observable{RGBAf}()
     map!(compute_lowcolor, c, lowcolor, c.extendlow, c.colormap)
     c.attributes[:_computed_extendlow] = lowcolor
-    is_extended_low = lift(!isnothing, c, lowcolor)
+    is_extended_low = lift(!is_zero_color, c, lowcolor)
```

Comment:
I don't remember at all why this was written as it is. But it seems to me this should just replace `lowcolor` with `c.extendlow` because what you check with `is_zero_color` is just set dependent on that:

```julia
function compute_lowcolor(el, cmap)
    if isnothing(el)
        return RGBAf(0, 0, 0, 0)
```

Reply:
Let's try that.